plugins/utils/autosource.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  ...
+}:
+let
+  inherit (lib.nixvim) defaultNullOpts;
+  mkBoolInt = defaultNullOpts.mkEnum [
+    0
+    1
+  ];
+in
+lib.nixvim.vim-plugin.mkVimPlugin {
+  name = "autosource";
+  originalName = "vim-autosource";
+  package = "vim-autosource";
+  globalPrefix = "autosource_";
+
+  maintainers = [ lib.nixvim.maintainers.refaelsh ];
```

FYI: this can be: 
```suggestion
  maintainers = [ lib.maintainers.refaelsh ];
```

As the two are now actually aliases.

I won't block on this though, because we'll do a treewide cleanup at some point.